### PR TITLE
upgrade to java8, build fat jar of finagle-postgres tests

### PR DESCRIPTION
**Pull request by dt**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,3 +29,5 @@ RUN pip install psycopg2
 RUN npm install pg
 
 RUN curl -SL https://jdbc.postgresql.org/download/postgresql-9.4.1207.jar > /postgres.jar
+
+RUN git clone https://github.com/cockroachdb/finagle-postgres.git && cd finagle-postgres && git checkout fatjartests && ./sbt assembly && mv target/scala-2.11/finagle-postgres-tests.jar /
```
Comment (tbg) [inline Dockerfile]: I _think_ no newline at EOF is non-standard.

Comment (dt) [inline Dockerfile]: fixed.

**Comment by tbg** [comment]
LGTM

File: Dockerfile (hunk 1 above)
Comment (tamird) [inline Dockerfile]: you can `git clone --depth 1 -b fatjartests` and save some width here

**Pull request merged**